about.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, height=device-height, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/featuredCourse.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <link rel="stylesheet" href="css/animations.css" />
+    <title>The Code Chaser</title>
+  </head>
+  <body>
+    <!-- #### header-section #### -->
+    <header>
+        <div class="header">
+          <a class="header-menu" href="#"><i class="fab fa-facebook-f"></i></a>
+          <a class="header-menu" href="#"><i class="fab fa-twitter"></i></a>
+          <a class="header-menu" href="#">English</a>
+          <a class="header-menu" href="#">My Page</a>
+          <a class="header-menu" href="#">Logout</a>
+        </div>
+        <div class="header2">
+      <a href="index.html"><img src="images/header-logo.png" alt="header logo" class="header-logo" /></a>
+
+      <!-- #### navbar-section #### -->
+      <nav>
+        <div class="desktop-nav">
+          <ul class="desktop-menu-items">
+            <li>
+            <a class="desktop-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Tutors</a>
```

Comment:
- [ ] The `header` is broken on the smallest desktop screen, please update it so it look perfect on all screen sizes.

Reference Design
![image](https://user-images.githubusercontent.com/33767951/142398362-1701891f-cc37-4373-9554-58e2ebf381b5.png)

Your Styling => `786px`
![image](https://user-images.githubusercontent.com/33767951/142398199-ebaa48f8-47ff-4700-a817-e198edfb336a.png)